cmd/clusterctl/client/repository/overrides.go

```diff
@@ -37,7 +35,7 @@ const (
 
 // Overrider provides behavior to determine the overrides layer.
 type Overrider interface {
-	Path() string
+	Path() (string, error)
```

Comment:
Let's document this in breaking changes for 1.3

Reply:
Good point, fixed